apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -572,7 +587,7 @@ input {
     padding: 0 15px;
 }
 
-.custom-tooltip {
+.custom-tooltipe {
```

Comment:
This change is reverted while according to the code design, we've fixed all the spelling mistakes. @prateek-giddh

Reply:
Done